### ISSUE TITLE
great project

### ISSUE THREAD

**Issue by jjhesk** [wo1fsea/yapre#76]
Can have more supports with a documentation?

Does it support webassembly in your plan?

**Comment by wo1fsea** [comment]
yes, yapre can now be build with emscripten to webassembly. see https://yapre.run/.
and for the documentation, https://docs.yapre.run/ is in progressing. 
but i am still refactoring almost everything, so docs may not be fininish in a short time.
